Resources/config/request_body_param_converter.xml

```diff
@@ -8,10 +8,10 @@
 
         <service id="fos_rest.converter.request_body" class="FOS\RestBundle\Request\RequestBodyParamConverter">
             <argument type="service" id="fos_rest.serializer"/>
-            <argument>%fos_rest.serializer.exclusion_strategy.groups%</argument>
-            <argument>%fos_rest.serializer.exclusion_strategy.version%</argument>
+            <argument /> <!-- serializer exclusion strategy groups -->
```

Comment:
shouldn't this be a collection rather than an empty string when loading the file ?